taskt/Core/Automation/Commands/Word Commands/WordAppendTextCommand.cs

```diff
@@ -1,137 +1,145 @@
-        [Attributes.PropertyAttributes.InputSpecification("Enter the unique instance name that was specified in the **Create Word** command")]
-        [Attributes.PropertyAttributes.SampleUsage("**myInstance** or **wordInstance**")]
-        [Attributes.PropertyAttributes.Remarks("Failure to enter the correct instance name or failure to first call **Create Word** command will cause an error")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyDescription("Word Instance Name")]
+        [InputSpecification("Enter the unique instance that was specified in the **Create Application** command.")]
+        [SampleUsage("MyWordInstance || {vWordInstance}")]
+        [Remarks("Failure to enter the correct instance or failure to first call **Create Application** command will cause an error.")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         public string v_InstanceName { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please Enter the Text Variable Name to Set")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Enter the text value that will be set.")]
-        [Attributes.PropertyAttributes.SampleUsage("Hello World or [vText]")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Text")]
+        [InputSpecification("Enter the text to append to the Document.")]
+        [SampleUsage("Hello World || {vText}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         public string v_TextToSet { get; set; }
 
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Select or Enter the text font name")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Arial")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Calibri")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Helvetica")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Times New Roman")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Verdana")]
-        [Attributes.PropertyAttributes.InputSpecification("Specify the font name.")]
-        [Attributes.PropertyAttributes.SampleUsage("Select **Arial**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Font Name")]
+        [PropertyUISelectionOption("Arial")]
+        [PropertyUISelectionOption("Calibri")]
+        [PropertyUISelectionOption("Helvetica")]
+        [PropertyUISelectionOption("Times New Roman")]
+        [PropertyUISelectionOption("Verdana")]
+        [InputSpecification("Select or provide a valid font name.")]
```

Comment:
I think It should simply be like "Select a font name."

Reply:
The thing is that these aren't the only options. You can use any font provided by Word. I just added the most common ones.

Reply:
got it